README.md

```diff
@@ -56,6 +56,10 @@ sink:
 pipeline:
   name: mysql-sync-doris
   parallelism: 4
+
+flink-config:
+  execution.checkpointing.interval: 30000
+  execution.checkpointing.mode: EXACTLY_ONC
```

Comment:
We need discuss the user facing API in dev group first and then open a PR, we should be careful when introduce new API for end users.

Reply:
@leonardBang Thank you for review. I have supplemented the relevant issues accordingly #2854.